migrations/2019-08-08-191129_add_users_public/up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE users ADD COLUMN public BOOLEAN NOT NULL DEFAULT 'f';
+UPDATE users
+SET public = 't'
+WHERE
+    id IN (
+        SELECT owner_id AS user_id FROM crate_owners WHERE NOT deleted AND owner_kind = 0
+        UNION
+        SELECT published_by as user_id FROM versions
```

Comment:
Is this information otherwise publicly available? We need it for DMCA purposes, but I'd rather not include it in the dump if it can't already be obtained through the API

Reply:
The `versions.published_by` column is publicly available, see e.g. https://crates.io/api/v1/crates/rand/0.7.0. The `versions_published_by` _table_, on the other hand, is private, and is marked as private in the configuration.